src/git.rs

```diff
@@ -134,10 +134,10 @@ pub fn tag(config: &Config, tag_name: &str, tag_message: &str) -> Result<(), Err
 
 pub fn push(config: &Config, tag_name: &str) -> Result<(), Error> {
     let repo      = &config.repository;
-    let token     = config.gh_token.as_ref().unwrap();
+    let token     = &config.gh_token.as_ref().unwrap();
```

Comment:
Because we're using `.as_ref` here, we already get a `&String` from that `unwrap`, we don't need to take another reference on that (with your code `token` is a `&&String`, which causes no problems, because it derefs to `&str` later in either case)